src/static/js/modules/per-page-js-loader.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var jsManifest = require( '../config/js-manifest-config.js' );
+
+/**
+ * Load a page-specific JavaScript dynamically into the header
+ * if the page has a page-specific script available.
+ */
+function init() {
+  var jsLoader = require( '../modules/util/js-loader' );
```

Comment:
Why define `jsManifest` outside the function and `jsLoader` inside the function?